lib/helpers/pack_hoodie.js

```diff
@@ -105,20 +106,21 @@ module.exports = function (env_config) {
   // Create the `bundleConcat` that we'll use to create the `browserify` bundle.
   fs.writeFileSync(bundleConcat, bundleTmpl + concatPlugins(plugins));
 
-  var b = browserify(bundleConcat);
-
-  return b.bundle({
+  var b = browserify(bundleConcat, {
     standalone: 'Hoodie',
     external: 'jquery'
-  }, function (err, bundle) {
+  });
+
+  b.bundle(function (err, bundle) {
     if (err) { return console.error(err); }
     // Store successful bundle in cache.
     cache = bundle;
     // Clean up! Remove dynamically generated file with concatenated scripts
     // once we are done building the bundle.
-    fs.unlink(bundleConcat, function (err) {
+    fs.unlinkSync(bundleConcat, function (err) {
```

Comment:
yeah maybe this would be better

```diff
-    fs.unlinkSync(bundleConcat, function (err) {
-      if (err) { return console.log(err); }
-    });
-    done(bundle);
+   fs.unlinkSync(bundleConcat, function (err) {
+     if (err) { return console.error(err); }
+     done(bundle);
+   });
```

Also `done(bundle)` looks wrong, just want to make sure, shouldn't that be `done(null, bundle)`?